superset/assets/javascripts/explore/components/PreviewChartPanel.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import $ from 'jquery';
+import { List } from 'immutable';
+import { Table } from 'reactable';
+import { withTableAutoSizer } from '@data-ui/data-table';
```

Comment:
Chris uses react-virtualized AutoSizer in [data-table](https://github.com/williaster/data-ui/blob/master/packages/data-table/src/enhancers/withTableAutoSizer.jsx), so you can use react-virtualized to avoid adding another js package.